src/wp-admin/includes/update-core.php

```diff
@@ -1538,12 +1538,12 @@ function update_core( $from, $to ) {
  *
  * @global array              $_old_requests_files Requests files to be preloaded.
  * @global WP_Filesystem_Base $wp_filesystem       WordPress filesystem subclass.
- * @global string             $wp_version          The WordPress version string.
  *
  * @param string $to Path to old WordPress installation.
  */
 function _preload_old_requests_classes_and_interfaces( $to ) {
-	global $_old_requests_files, $wp_filesystem, $wp_version;
+	global $_old_requests_files, $wp_filesystem;
+	$wp_version = wp_get_wp_version();
 
 	/*
```

Comment:
Hey @peterwilsoncc, should the global also be changed in `_redirect_to_about_wordpress()`?

Reply:
Yep, done in 1c9ed13f2ca47a57551045a03904c1b4874126ea.

Thanks for the review.